yal/onboarding.py

```diff
@@ -287,9 +290,80 @@ async def state_sexual_literacy_assessment_end(self):
                 "🤦🏾‍♂️ Thanks for being so patient and honest 😌.",
             ]
         )
-        return EndState(self, msg)
 
-        # TODO: Go down different path depending on assessment outcome
+        await self.publish_message(msg)
+        await asyncio.sleep(0.5)
+        score = self.user.metadata.get("assessment_score", 0)
+        if score <= 25:
+            # score of 0-25 high risk
+            risk = "high_risk"
+        else:
+            # score of 25-30 low risk
+            risk = "low_risk"
+
+        msisdn = normalise_phonenumber(self.inbound.from_addr)
+        whatsapp_id = msisdn.lstrip(" + ")
+        data = {
+            "sexual_health_lit_risk": risk,
+        }
```

Comment:
I'm wondering if we should be storing the score on the contact field? Otherwise right now it's lost at the end of this session. That's what I've got for the default behaviour of the other assessments, storing just the score on the contact field.